CHANGELOG-nightly.md

```diff
@@ -4,6 +4,7 @@
 
 -   Links in chat messages now respect known TLDs instead of matching any url-like pattern
 -   Added an option to show timeouts/bans directly in the chat without being a moderator
+-   Add option to hide suspicious user highlight
```

Comment:
This should probably be "Added an option to hide suspicious user highlights" to match with the rest of the changelog